samples/grids/tree-grid/data-searching/BlazorClientApp.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <RazorLangVersion>3.0</RazorLangVersion>
+    <AssemblyName>Infragistics.Samples</AssemblyName>
+    <RootNamespace>Infragistics.Samples</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702,IDE0028,BL0005,0219,CS1998</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="IgniteUI.Blazor" Version="23.2.204" />
```

Comment:
Update **IgniteUI.Blazor** Package Version to **23.2.218**.

`<PackageReference Include="IgniteUI.Blazor" Version="23.2.218" />`